{"uuid":"0c5aaced-af9e-40d6-8fd4-084eee1262bc","historyId":"cf235ef5167cd0b6094cbc03f6199248","testCaseId":"[engine:junit-jupiter]/[class:com.example.data.repository.RevisionRepositoryTest]/[method:insert should throw exception when dao fails()]","testCaseName":"insert should throw exception when dao fails()","fullName":"com.example.data.repository.RevisionRepositoryTest.insert should throw exception when dao fails","labels":[{"name":"junit.platform.uniqueid","value":"[engine:junit-jupiter]/[class:com.example.data.repository.RevisionRepositoryTest]/[method:insert should throw exception when dao fails()]"},{"name":"host","value":"Maksims-MacBook-Air.local"},{"name":"thread","value":"79128@Maksims-MacBook-Air.local.Test worker(1)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"com.example.data.repository.RevisionRepositoryTest"},{"name":"testClass","value":"com.example.data.repository.RevisionRepositoryTest"},{"name":"testMethod","value":"insert should throw exception when dao fails"},{"name":"suite","value":"com.example.data.repository.RevisionRepositoryTest"}],"links":[],"name":"insert should throw exception when dao fails()","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1736938963092,"stop":1736938963098}